src/resources.py

```diff
@@ -114,6 +117,10 @@ def list_resources(label, label_value, target_folder, request_url, request_metho
     if script and files_changed:
         execute(script)
 
+    if request_ignore_initial_event and ignore_request:
```

Comment:
This gets executed after processing all the items in the batch. Only if a single one hasn't been seen before, we skip executing the request, despite other already known items might have received updates.

The issue here is that we're doing the request only once for a batch whenever at least a single item changed. Following this logic, `request_ignore_initial_event` should only count as flag to determining whether something has changed, i.e. only when *all* items in the batch receive their initial version or an insignificant update (no changes to content), the request gets skipped.

Reply:
Thanks, you're correct. I inverted the bool to make sure it works properly for batching items.